deploy/helm/kubecf/assets/operations/sequencing.yaml

```diff
@@ -0,0 +1,125 @@
+# We specify the new dependencies (I) and remove the old serialization
+# hints (II).
+#
+# I. Custom pod dependencies for startup sequencing
+##
+# ATTENTION: Annotations are strings. While the cf-operator's
+# controller will unmarshall the value as a json array later on,
+# specification must be done as a string using json syntax inside.
+##
+# Short form of the ops changes below.
+# Dependencies pointing to dependents.
+# Order here matches order of ops instructions.
+##
+# auctioneer <-- diego-api
+# scheduler  <-- api
+# cc-worker  <-- api
+# router     <-- routing-api <-- uaa
+#                tcp-router  <-- uaa
+#                log-cache   <-- uaa
+#                credhub     <-- uaa
+#                diego-cell  <-- uaa
+#
+# asactors  <- asdatabase
+# asapi     <- asdatabase
+# asmetrics <- asdatabase
+# asnozzle  <- asdatabase
+##
+
+- type: replace
+  path: /instance_groups/name=auctioneer/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
```

Comment:
This fails in Eirini mode because that doesn't have an auctioneer.

Reply:
Fixed with commit c1208bae851e70c35237bf61d0dd16f2f5dbd20a